packages/builder/src/schematics/ng-add/template/env.d.ts

```diff
@@ -22,6 +22,7 @@ interface ImportMetaEnv {
  * @deprecated process.env usage
  * prefer using import.meta.env
  * */
+// @ts-ignore
```

Comment:
I'd suggest to remove the whole process.env thing and push for import.meta.env usage
or we can add it as commented